File: 1 - Sequenciais/sequenciais9.java
//Faça um programa que peça a temperatura em graus Farenheit, transforme e mostre a temperatura em graus Celsius.
//C = (5 * (F-32) / 9)

import java.util.Scanner;

public class sequenciais9 {

  public static void main(String[] args) {
    System.out.println("Digite uma temperatura, em Farenheit: ");
    float farenheit = new Scanner(System.in).nextFloat();
    double celsius = (5 * (farenheit - 32)) / 9;
    System.out.printf("A temperatura, em Celsius, eh de: %.2f °C", celsius);
  }

}

File: 1 - Sequenciais/sequenciais9.php
<?php

#Faça um programa que peça a temperatura em graus Farenheit, transforme e mostre a temperatura em graus Celsius.
#C = (5 * (F-32) / 9)

?>

<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Sequenciais 9 - PHP</title>
  </head>
  <body>
    <?php if (!empty($_GET)): ?>
      <?= 'A temperatura, em Celsius, é de: ' . ((5 * ($_GET['farenheit'] - 32)) / 9); ?>
    <?php else: ?>
      <form action="sequenciais9.php" method="get">
        Digite uma temperatura, em Farenheit: <input type="text" name="farenheit"/><br>
        <input type="submit" value="Enter">
      </form>
    <?php endif; ?>
  </body>
</html>
